package.json

```diff
@@ -15,11 +15,13 @@
     "@testing-library/user-event": "^13.5.0",
     "axios": "^0.26.1",
     "jwt-decode": "^3.1.2",
+    "moment": "^2.29.4",
     "react": "^18.0.0",
     "react-dom": "^18.0.0",
     "react-router-dom": "^6.3.0",
     "react-scripts": "5.0.1",
     "styled-components": "^5.3.5",
+    "typescript": "^4.8.3",
```

Comment:
Why are we adding typescript here?

Reply:
I was reading the `moment.js` documentation, the one she used, and apparently they are using typescript for some of the things you can do, like `moment().format()`. I think that is why it is there. 
I'm reading this documentation just now so I'm not 100% sure.
<img width="772" alt="Screenshot 2022-09-23 at 14 54 07" src="https://user-images.githubusercontent.com/94084046/191976559-892352ec-bf04-4904-97cd-9b283796d739.png">

Reply:
Ah, that's just saying that _if_ you were using typescript, moment 2.13.0 now includes a type file for the library. We don't need to add typescript to the project :)

Reply:
I installed it because cypress does not let me pass unless I don't install Typescript